kustomize/envs/Development/variables.properties

```diff
@@ -0,0 +1,5 @@
+DATABASE_HOSTNAME=#{DATABASE_HOSTNAME}

```

Comment:
Is there any reason these variables couldnt follow the `.` notation used for other variable substitution? 

e.g. `#{Project.Database.Hostname}`